.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - checkout
       - run:
           name: Update npm
-          command: 'npm install -g npm@latest'
+          command: 'sudo npm install -g npm@latest'
```

Comment:
Requires `sudo` perms.  Will not work without ...

Reply:
Agreed matches up with the template project toohttps://github.com/ministryofjustice/hmpps-template-typescript/blob/main/.circleci/config.yml